src/components/cover/cover.types.ts

```diff
@@ -0,0 +1,48 @@
+import { Cover } from "../../core/dbc-gateway/generated/graphql";
+
+/**
+ * The keys in the GraphQL `Cover` object that refer to the actual image sizes.
+ * Present in the data.
+ */
+export type FbiCoverImageSizeKey = Extract<
+  keyof Cover,
+  "xSmall" | "small" | "medium" | "large"
+>;
```

Comment:
I appreciate this level of commenting.

I think it is important for the reader to get an understanding of why our custom types exist.